src/main/java/net/snowflake/ingest/streaming/internal/BlobBuilder.java

```diff
@@ -319,7 +319,7 @@ static String computeMD5(byte[] data, int length) throws NoSuchAlgorithmExceptio
         MessageDigest.getInstance("MD5");
     md.update(data, 0, length);
     byte[] digest = md.digest();
-    return DatatypeConverter.printHexBinary(digest).toLowerCase();
+    return Hex.encodeHexString(digest);
```

Comment:
Is the result always the same by these two functions?

Reply:
Yes, I verified that they are the same, `Hex.encodeHexString(String)` returns lower-case hexadecimal string.